backend/src/test/java/zipgo/domain/KeywordTest.java

```diff
@@ -0,0 +1,32 @@
+package zipgo.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class KeywordTest {
+    @Test
+    void 이름으로_Keyword_를_생성할_수_있다() {
+        // given
+        String name = "다이어트";
+
+        // when
+        Keyword keyword = new Keyword(name);
+
+        // then
+        assertThat(keyword.getName()).isEqualTo("다이어트");
+    }
+
+    @Test
+    void 아이디가_같으면_동등하다() {
+        //given
+        Keyword 키워드_1 = new Keyword(1L, "다이어트");
+        Keyword 키워드_2 = new Keyword(1L, "다이어트");
+
+        //when
+        boolean 동등함 = 키워드_2.equals(키워드_1);
+
+        //then
+        assertThat(동등함).isTrue();
```

Comment:
```suggestion
        // expect
        assertThat(키워드_1).isEqualTo(키워드_2);
```

when, then을 합쳐서 작성하시는 건 어떻게 생각하시나요?

Reply:
합쳐서 작성하는 거 전 좋아요~ 다만 제가 given-when-then에 익숙해서 앞으로도 놓칠순 있을 거같아요 ㅜ ㅋㅋ
 다른 분들 의견 궁금합니다 @wonyongChoi05 @parkmuhyeun

Reply:
이게 가끔 합칠 상황이 나오는거라 꼭 합치자는 것은 아니구,,,
내일 같이 얘기 나눠보면 좋을 것 같아요~

Reply:
저도 상황에 따라 붙여 쓰는거 좋아하는 편이에요!